Core/GDCore/Extensions/Builtin/BaseObjectExtension.cpp

```diff
@@ -550,7 +550,7 @@ void GD_CORE_API BuiltinExtensionsImplementer::ImplementsBaseObjectExtension(
                    _("Z-order"),
                    _("Compare the Z-order of the specified object."),
                    _("the Z-order"),
-                   _("Z-order"),
+                   _("Layer"),
```

Comment:
Why isn't it the same group as above?

Reply:
The one for actions is not the same as for conditions. I wanted to change them but I fear to have conflict so, I'll do it later.